test/e2e/utils/utils.go

```diff
@@ -62,9 +63,10 @@ func GCEClientAndDriverSetup(instance *remote.InstanceInfo, driverConfig DriverC
 		fmt.Sprintf("--extra-labels=%s=%s", DiskLabelKey, DiskLabelValue),
 		"--max-concurrent-format-and-mount=20", // otherwise the serialization times out the e2e test.
 		"--multi-zone-volume-handle-enable",
-		"--multi-zone-volume-handle-disk-types=pd-standard",
+		"--multi-zone-volume-handle-disk-types=pd-standard,hyperdisk-ml",
```

Comment:
I thought it was pd-ssd instead of pd-standard?

Reply:
This is just a testing configuration. I don't think anyone would want to enable --multi-zone-volume-handle-disk-types on `pd-standard`. I can update the tests?